src/pages/HomePage/HomePage.js

```diff
@@ -1,37 +1,30 @@
 import styled from 'styled-components/macro'
 import Header from './../../components/Header/Header'
+import Navigation from '../../components/Navigation/Navigation'
 
-export default function HomePage() {
+export default function HomePage({ pages }) {
```

Comment:
So this component only renders a header an navigation? Not quite sure why 🙂

Reply:
In this user story yes, but I want to add a slider with pictures soon and maybe a weather api. I hope I can get it all done! :) Thank you for reviewing. :) 👍